.github/workflows/ci.yaml

```diff
@@ -111,7 +111,7 @@ jobs:
           export GOPATH=$(go env GOPATH)
           export PATH=${PATH}:$GOPATH/bin
           make verify lint binary test test/integration
-        timeout-minutes: 14
+        timeout-minutes: 20
```

Comment:
Why 20?

Reply:
Just a guess because 14 failed :) 

It looks like there are a lot more tests now like for kafka auth, it's timing out in 20 mins while just running kafka tests. It hasn't gotten to connector tests yet, see https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/runs/5475757655?check_suite_focus=true

Reply:
@machi1990 FYI - maybe someone should look into figuring out how run more kafka tests in parallel to reduce total test time.

Reply:
> It looks like there are a lot more tests now like for kafka auth

@dhirajsb Hmm I think the test were timing out for a different reason. And this has since been fixed https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/840, so I'd advise rebasing on main if you haven't done so.

 > @machi1990 FYI - maybe someone should look into figuring out how run more kafka tests in parallel to reduce total test time.

+1 true, but you cannot really run parallel integration tests with a shared database that's get's cleanup / reset after each test. You are only opening room for some racey conditions. As I said above the the test timeout issue was more to do with the CI secrets setup (fixed in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/840) rather than the tests being many as the problem would have occurred in either way the tests are run.

Reply:
That makes sense. I looked at older PR that this is a copy of, and tests were taking 8mins earlier. I'll revert this change, rebase and push again.

Reply:
The CI issue is resolved, integration tests are working again.

Reply:
Thanks @dhirajsb